demisto_sdk/commands/common/logger.py

```diff
@@ -2,28 +2,91 @@
 import logging
 import logging.config
 import os.path
+import platform
 import re
 import sys
 from logging.handlers import RotatingFileHandler
 from pathlib import Path
 from typing import Dict, List, Optional, Union
 
-from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
-from demisto_sdk.commands.common.tools import parse_int_or_default, string_to_bool
+# NOTE: Do not add internal imports here, as it may cause circular imports.
```

Comment:
You mean "Don't import anything other than a constant", right?
This comment isn't clear.

Reply:
It says "Do not add internal imports here", as in adding additional ones to the existing one there.

But yes, that's what I meant.